kolibri/core/assets/src/views/Navbar/NavbarLink.vue

```diff
@@ -68,6 +70,9 @@
           borderBottomWidth: '2px',
         };
       },
+      isActive() { 
+        return this.$route.path === `/${this.link.name.toLowerCase()}`;
```

Comment:
We can't rely on this comparison to determine whether a link is active.

`link.name` contains our internal route name which in some cases may indeed correspond to the actual URL, but that happens rather by chance.

One place where you can see such solution fails is in _"Coach" --> "Class home"_ tab. Text of the active _"Class home"_ tab is `rgb(250, 250, 250)` (`this.$themePalette.grey.v_50`) instead of expected `rgb(255, 255, 255)` (`this.$themeTokens.textInverted`). Even though the tab is active, `isActive` is evaluated to `false` due to `link.name` being `HomePage`whereas the actual URL contains only `home`. In some cases, it is even slightly more complicated as `$route.path` may contain some more information, e.g. a class ID.

Looking into `$route.name` won't help us either, as again in some cases, like when sub-routing, it is different from `link.name`. You can, for example, log both values in _"Coach" -> "Plan" -> "Quizzes"_ to see it.